src/main/java/graphql/execution/ExecutionStrategy.java

```diff
@@ -766,8 +765,8 @@ protected ExecutionStepInfo createExecutionStepInfo(ExecutionContext executionCo
         GraphQLOutputType fieldType = fieldDefinition.getType();
         Field field = parameters.getField().get(0);
         List<Argument> fieldArgs = field.getArguments();
-        GraphqlFieldVisibility fieldVisibility = executionContext.getGraphQLSchema().getFieldVisibility();
-        Map<String, Object> argumentValues = valuesResolver.getArgumentValues(fieldVisibility, fieldDefinition.getArguments(), fieldArgs, executionContext.getVariables());
+        GraphQLCodeRegistry codeRegistry = executionContext.getGraphQLSchema().getCodeRegistry();
+        Map<String, Object> argumentValues = valuesResolver.getArgumentValues(codeRegistry, fieldDefinition.getArguments(), fieldArgs, executionContext.getVariables());
 
         return newExecutionStepInfo()
                 .type(fieldType)
```

Comment:
Most of these changes are in fact moving away from the deprecated `graphqlSchema.getFieldVisibility()`